custom_components/extended_openai_conversation/__init__.py

```diff
@@ -277,6 +277,7 @@ def get_exposed_entities(self):
                     "entity_id": entity_id,
                     "name": state.name,
                     "state": self.hass.states.get(entity_id).state,
+                    "lastUpdated": self.hass.states.get(entity_id).last_updated,
```

Comment:
Could you change `lastUpdated` to `last_updated`?
Since it uses `entity_id` above, I want to keep snake case as a convention for exposed entity.

```suggestion
                    "last_updated": self.hass.states.get(entity_id).last_updated,
```